fiftyone/core/utils.py

```diff
@@ -3206,19 +3206,76 @@ def async_executor(
         Exception: if a task raises an exception and ``skip_failures == False``
     """
     with ThreadPoolExecutor(max_workers=max_workers) as executor:
-        _futures = []
+        futures = []
 
         def submit(*args, **kwargs):
             future = executor.submit(*args, **kwargs)
-            _futures.append(future)
+            futures.append(future)
             return future
 
         yield submit
 
-        for future in _futures:
+        for future in futures:
```

Comment:
These lines are an opportunistic cleanup in the original code.

With this branch this function is no longer used in fiftyone, so it could be removed, but the code has been released so there could be third-party uses of it. Suggestions on whether to keep it or get rid of it?